docs/_layouts/foam.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+---
+
+<script type="text/javascript">
+var MD_EXT = ['.md', '.markdown', '.mdown', '.mkdn', '.mkd', '.mdwn', '.mdtxt', '.mdtext', '.text', '.Rmd'];
```

Comment:
Oh yes, this was in a html-file used for publishing, so cannot really easily use package.json. In perfect world we would have tooling to read this out from the package.json and output it here (or similar mechanics). For now I think this is quite good. One thing we still should change - not necessarely in this PR though - is the file extension use in the vscode-extension. I was able to find at least this line:

```typescript
(f) => f.scheme === "file" && f.path.match(/\.(md|mdx|markdown)/i)
```

that should be aligned as well. I would expect that line to be constructed based on the user settings. For this MD_EXT fragment I'd add a disclaimer to synchronize the file extensions with the other places in the codebase.

Reply:
sounds good, will add the disclaimer